src/Compilers/CSharp/Test/Syntax/Parsing/DeclarationParsingTests.cs

```diff
@@ -7935,7 +7935,7 @@ class Box
         [Fact]
         public void TestAnonymousDelegateNullChecking()
         {
-            // PROTOTYPE : During semantics, make sure this causes an error
+            // https://github.com/dotnet/roslyn/issues/58323
```

Comment:
Did not see a test for `delegate` parameters in the branch. Tried a couple of searches but could've missed it. Verified manually though that the correct behavior is in place.

Reply:
https://github.com/dotnet/roslyn/blob/e5e5d2108cc54355a6a4c51e952babed4d5d51ee/src/Compilers/CSharp/Test/Semantic/Semantics/NullCheckedParameterTests.cs#L21-L33

Reply:
ugg, how did I miss that????